cms/test_utils/project/placeholderapp/views.py

```diff
@@ -24,7 +25,12 @@ def _base_detail(request, instance, template_name='detail.html',
         request.toolbar.set_object(instance)
     if template_string:
         template = Template(template_string)
-        return HttpResponse(template.render(RequestContext(request=request, dict_=context)))
+        if DJANGO_1_7:
+            return HttpResponse(template.render(RequestContext(request=request, dict_=context)))
+        else:
+            from django.template.context import make_context
+            context = make_context(context, request)
+            return HttpResponse(template.render(context))
```

Comment:
testsuite is riddled with private API usage to test things at low level.
Calling ``template.render(RequestContext)`` trigger this deprecation https://github.com/django/django/blob/stable/1.9.x/django/template/backends/django.py#L82
Using abstract ``django.template.base.Template`` + ``make_context`` results in much cleaner code than using ``django.template.backends.django.Template``